common/scala/src/main/scala/org/apache/openwhisk/core/database/cosmosdb/CosmosDBArtifactStore.scala

```diff
@@ -78,7 +78,7 @@ class CosmosDBArtifactStore[DocumentAbstraction <: DocumentSerializer](protected
     this,
     s"Initializing CosmosDBArtifactStore for collection [$collName]. Service endpoint [${client.getServiceEndpoint}], " +
       s"Read endpoint [${client.getReadEndpoint}], Write endpoint [${client.getWriteEndpoint}], Connection Policy [${client.getConnectionPolicy}], " +
-      s"Time to live [${collection.getDefaultTimeToLive} secs, clusterId [${config.clusterId}], soft delete TTL [${config.softDeleteTTL}]")
+      s"Time to live [${collection.getDefaultTimeToLive} secs, clusterId [${config.clusterId}], soft delete TTL [${config.softDeleteTTL}], Consistency Level [${config.consistencyLevel}]")
```

Comment:
is there a default value for `config.consistencyLevel` ?

Reply:
yes it is here: https://github.com/apache/incubator-openwhisk/blob/e7253ac5bac519df289321a22c5ee923bb96c1f7/common/scala/src/main/resources/application.conf#L194